pallets/nfts/src/tests.rs

```diff
@@ -2188,6 +2885,223 @@ fn clear_all_transfer_approvals_works() {
 	});
 }
 
+#[test]
+fn clear_collection_approvals_works() {
```

Comment:
Test origin.

This should also be able to be combined into one test with `force_`

Reply:
Resolved by combining this test with `force_clear_collection_approvals_works`. Same for `approve_collection_approval_works` and `clear_collection_approval_works`